.github/workflows/push_pr.yaml

```
@@ -76,7 +76,7 @@ jobs:
       - name: Codespell test
         uses: codespell-project/actions-codespell@master
         with:
-          skip: ./.git
+          skip: ./.git,internal/testutil/data
```

🔝

---

didn't work 😭

---

Seems like codespell is ignoring it though: https://github.com/newrelic/nri-kubernetes/runs/4562163854?check_suite_focus=true#step:4:20
From the [docs](https://github.com/codespell-project/actions-codespell#parameter-skip) it says it expects a glob, perhaps we have to use that syntax?
```suggestion
          skip: ./.git,internal/testutil/data/**
```

---

i'll give a try

---

working know , the action interface doesn't work as expected. I just added the local make target and `.codespellrc` with de config, so we can run locally and is "the same" as the ci runs